presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
+  so INNER join should not have PARTITIONED distribution type".
+* Add support for recoverable grouped execution. This can be enabled with the
+  ``recoverable_grouped_execution`` session property.
+* Add support for grouped execution for ScanFilterProject-only queries. This
+  can be enabled with the ``grouped_execution_for_eligible_table_scans`` session
+  property.
+* Add config property ``max-concurrent-materializations`` and session property
+  ``max_concurrent_materializations`` to limit the number of plan sections that
+  will run concurrently when using materialized exchanges.
+* Add support for computing :func:`approx_distinct` over BingTile values.
+* Add :func:`merge_hll` which performs a scalar merge of an array of HyperLogLogs.
+* Add bitwise shift operations, :func:`bitwise_arithmetic_shift_right`,
+  :func:`bitwise_logical_shift_right` and :func:`bitwise_shift_left`.
+* Add completed and total lifespans on Web UI.
+
+Hive Connector Changes
+----------------------
+* Fix failures for ``information_schema`` queries when a table has an invalid storage format.
+* Improve query execution time over bucketed table with large buckets.
+* Add config property ``hive.metastore.glue.catalogid`` to configure the Glue catalog ID.
+
+Bytecode Changes
+----------------
+* Restore bytecode library deleted in version 0.193.
+
+SPI Changes
+-----------
+* Add experimental interface ``ConnectorPlanOptimizer`` to allow connectors to
```

Comment:
It would be nice to link to an issue or design doc

Reply:
That is a good point. Even I feel that adding details in the release notes is not the best way to talk about the feature.

Reply:
@highker - can you point me to the issue ?

Reply:
https://github.com/prestodb/presto/pull/12920/commits/fd6ae44d37984dd37d0c3f8055ec435cbb01975b#diff-96c6cc2dde994497e64c36f816baf0c2R23 is the javadoc. That is the only thing we have for now.

Reply:
I agree this a big change; documentation is desired. Actually, if there is a way to link to the design doc in slack (https://prestodb.slack.com/archives/C07JH9WMQ/p1555713637042300), that would be good too.

Reply:
why don't you copy the design doc into a github issue?

Reply:
Quickly created on (#13102). Hope that could help